packages/middleware/src/handlers/prepareApiFunction/index.ts

```diff
@@ -76,19 +74,14 @@ export function prepareApiFunction(
     });
 
     // Pick the function from the namespaced if it exists, otherwise pick it from the shared integration
-    try {
-      res.locals.apiFunction = getApiFunction(
-        apiClientInstance,
-        functionName,
-        extensionName
-      );
-    } catch (e) {
-      res.status(404);
-      res.send(e.message);
-
-      return;
-    }
-
-    next();
+    getApiFunction(apiClientInstance, functionName, extensionName)
```

Comment:
nit pick but I'd like to have async/await syntax in here as well (easier debugging imho), but you'd need to use this lib: https://www.npmjs.com/package/express-async-handler